tools/mkldscript.c

```diff
@@ -267,8 +270,11 @@ static void parse_rom_spec(char *spec)
                         util_fatal_error("line %i: alignment is not a power of two", lineNum);
                     break;
                 case STMT_include:
+                case STMT_include_readonly:
                     currSeg->includesCount++;
                     currSeg->includes = realloc(currSeg->includes, currSeg->includesCount * sizeof(*currSeg->includes));
+                    currSeg->readOnlyData = realloc(currSeg->readOnlyData, currSeg->includesCount * sizeof(*currSeg->readOnlyData));
+                    currSeg->readOnlyData[currSeg->includesCount - 1] = (stmt == STMT_include_readonly);
```

Comment:
I think `putDataWithRodata` may be a better name given what this achieves (and `include_readonly` -> `include_datawithrodata`, I know the name sucks but idk this whole thing looks bad anyway imo, so may as well make it explicit)

is there no way to actually move the relevant data from .data to .rodata ?

Reply:
It is possible to move the data from .data to .rodata by wrapping everything in const arrays of primitives, however this only gets us so far (doesn't work with the message tables) and is quite unreadable.

Reply:
I'm open to renaming this also but I'd like to wait a little to see if anyone has any other options for the name.